sechub-wrapper-prepare/src/main/java/com/mercedesbenz/sechub/wrapper/prepare/modules/skopeo/SkopeoLocationConverter.java

```diff
@@ -16,7 +16,7 @@ public String convertLocationForLogin(String location) {
         return removeProtocolPrefix(location);
     }
 
-    private String removeProtocolPrefix(String location) {
+    public String removeProtocolPrefix(String location) {
```

Comment:
The class is named SkopeoLocationConverter.

You made the private method here just public. but we use internally the public method again public->private->public ... 

Please do following:
- keep the private method as private.
- introduce a method "convertToAddititionalTag(...)" and ue this inside Skopeo wrapper. You can call the private part from there
- add some unit tests for convertToAddititionalTag
- add javadoc and describe on convertToAddititionalTag if the tag result could ever be empty or null (we add it inside commands list and I am currently not 100% sure what happens /if it is impossible, that we could have here null

so it is clear that we use something for tagging.
And it is tested.